src/js/lib/admin-page/index.js

```diff
@@ -1,7 +1,13 @@
 import domReady from '@wordpress/dom-ready';
 import { __, sprintf } from '@wordpress/i18n';
```

Comment:
Now that we're completely taking over the admin panel, we can pull out this line:
```suggestion
```

...& also the remainder of this function:
https://github.com/Parsely/wp-parsely/blob/171714f56d90a57f8385dfd74965d775fc5846cc/src/js/lib/admin-page/index.js#L10-L33